wiremock-spring-boot-example/src/test/java/app/HttpsTest.java

```diff
@@ -0,0 +1,60 @@
+package app;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.anyRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.anyUrl;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.wiremock.spring.ConfigureWireMock;
+import org.wiremock.spring.EnableWireMock;
+import org.wiremock.spring.InjectWireMock;
+
+@SpringBootTest
+@EnableWireMock({@ConfigureWireMock(useHttps = true)})
+class HttpsTest {
```

Comment:
I might be wrong but I thought you could have one wiremock instance with both http and https enabled. The current setup would force me to have two wiremock instances if I want to run on http and https at the same time. 

Having to use {{@ConfigureWireMock(port = 8081)}, @ConfigureWireMock(useHttps = true, port = 8082)}